bin/src/ctl/command.rs

```diff
@@ -59,10 +59,9 @@ impl CommandManager {
     ) -> anyhow::Result<()> {
         let command_request = CommandRequest::new(id.to_string(), command_request_order, None);
 
-        if !self.channel.write_message(&command_request) {
-            bail!("Could not write the request");
-        }
-        Ok(())
+        self.channel
+            .write_message(&command_request)
+            .with_context(|| "Could not write the request")
```

Comment:
Tu me, it miss a `?`

Reply:
Then it misses also a `;` and an `Ok(())` ! These are useless because `.write_message` and `send_request` have the same return type. 
But I can add all those anyway.